org/cloudfoundry.yml

```diff
@@ -2548,3 +2548,18 @@ orgs:
           terraform-provider-csbpg: admin
           terraform-provider-csbmysql: admin
           terraform-provider-csbsqlserver: admin
+      temp-uaa-ci-admins:
+        description: "Temporary team for managing UAA CI"
+        maintainers:
+          - coolgang123
+          - duanemay
+          - iprotsiuk
+        members: [ ]
+        privacy: closed
+        repos:
+          cf-uaac: admin
```

Comment:
Why are these team members also getting `admin` rights on these repo's? I'm assuming read would be enough? Or no repo's needed at all, since it would just be used to grant access in concourse.

Reply:
Pushed another commit.